src/main/scala/is/hail/utils/Graph.scala

```diff
@@ -0,0 +1,53 @@
+package is.hail.utils
+
+import scala.collection.mutable
+import scala.reflect.ClassTag
+
+object Graph {
+  def mkGraph[T](edges: (T, T)*): mutable.MultiMap[T, T] =
+    mkGraph(edges.toArray)
```

Comment:
would it be more efficient to traverse rather than make the array? You made that interface too

Reply:
varargs methods are compiled to create an `Array` that is put into a `WrappedArray` (which implements `Seq`), so `toArray` here is almost a no-op ([its a virtual method call](https://github.com/scala/scala/blob/v2.12.0/src/library/scala/collection/mutable/WrappedArray.scala#L67-L73) that contains a couple checks and a cast)